chpl/chpl-service/src/main/java/gov/healthit/chpl/standard/CertificationResultStandard.java

```diff
@@ -31,6 +32,9 @@ public class CertificationResultStandard implements Serializable {
     @JsonIgnore
     private Long certificationResultId;
 
+    @JsonIgnore
+    private Date createDate;
```

Comment:
I feel like everywhere else this is "creationDate". Any reason it's different or objection to changing it?